src/python/espressomd/visualization.py

```diff
@@ -1278,18 +1266,17 @@ def _draw_lb_vel(self):
     def _modulo_indexing(l, t):
         return l[t % len(l)]
 
-    # FADE PARTICLE CHARGE COLOR FROM WHITE (q=0) to PLUSCOLOR (q=q_max) RESP
-    # MINUSCOLOR (q=q_min)
+    # FADE PARTICLE CHARGE COLOR FROM WHITE (q=0) to PLUS_COLOR (q=q_max) RESP
```

Comment:
At some point, we should probably just convert the comments in `visualization.py` to normal capitalization instead of UPPERCASE...

Reply:
done! it took forever